src/Tribe/Views/V2/Views/Month_View.php

```diff
@@ -346,12 +346,13 @@ static function ( $event ) use ( $date_object ) {
 			);
 
 			$start_of_week = get_option( 'start_of_week', 0 );
+			$is_start_of_week = (int)$start_of_week === (int)$date_object->format( 'w' );
 
 			$day_url = tribe_events_get_url( [ 'eventDisplay' => 'day', 'eventDate' => $day_date ] );
 
-			$day_data = [
+			$day_data         = [
 				'date'             => $day_date,
-				'is_start_of_week' => $start_of_week === $date_object->format( 'w' ),
+				'is_start_of_week' => $is_start_of_week,
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: b471f4c91ed9e9a4441fbcd46e940644</sub>